drivers/sensor/bmi160/bmi160.h

```diff
@@ -376,6 +425,50 @@ enum bmi160_odr {
 #	define BMI160_DEFAULT_ODR_GYR		13
 #endif
 
+#if defined(CONFIG_BMI160_MAG_ODR_RUNTIME) ||\
```

Comment:
DeviceTree bindings options could be used instead of Kconfig options for defaults.

Reply:
@gmarull agree for new driver, here just followed same way existing bmi160.